### DISCUSSION TITLE
AM de-modulation only poor in Spectrum analyzer mode

### DISCUSSION THREAD

**Discussion by jh4vaj** [egzumer/uv-k5-firmware-custom#101] [Q&A]
It seems that the Spectrum analyzer mode in firmware v0.16 and v0.17 does not allow for AM (airband) demodulation. 
There are no issues with AM demodulation when not in Spectrum analyzer mode.

In v0.15, AM can be demodulated even in Spectrum analyzer mode.

Is there any other setting required for Spectrum analyzer mode in v0.16 and later?

**Comment by ErikS-web** [comment]
Do you mean AM demodulation not allowed in Spectrum analyzer

Can you be more specific. 
What is the sweep start frequency (VFO) when you start Spectrum analyzer.
So I can test.

Just press `0` - toggle modulation type (FM/AM/USB). What happens on screen ?

**Reply by jh4vaj** [reply]
Can switch demodulation mode (FM/AM/USB).
However, even when set to AM, v0.16 and v0.17 do not sound normal.
With v0.15, AM demodulation sounds normal audio.

The sweep start frequency is 118MHz and step is 12.5kHz, of course, demod is AM.

**Reply by egzumer** [reply]
There is no difference as far as I know between 15 and 16.

**Reply by jh4vaj** [reply]
I tried this with two radios.
One is UV-K5, and the other is UV-K5(8).
Both had the same result, that is, v0.15 was normal, but there were problems with V0.17.
It's strange.

**Reply by magumataishi** [reply]
I also had a problem with V0.16 and V0.17 not being able to demodulate AM correctly.
When receiving particularly strong radio waves, the voice cannot be heard.
I was able to confirm this with Airband 118MHz-120MHz AM.

**Reply by magumataishi** [reply]
I'm using UV-K6. It was normal until v0.15.
Note that in normal mode Airband AM is demodulated correctly.

**Reply by jh4vaj** [reply]
It works fine in v0.20.
Thank you very much.

**Comment by ErikS-web** [comment]
0.15 OK and later NOT ... [this are the differences between 0.15 and 0.16](https://github.com/egzumer/uv-k5-firmware-custom/compare/v0.15...v0.16)

**Reply by ErikS-web** [reply]
Modified title form "AM modulation" to AM modulation only poor in Spectrum analyzer mode.
Will transfer this ticket the Discussions to Issues

**Reply by ErikS-web** [reply]
Looks like ticket 113, will combine.

And continue as Issue, #118

**Comment by ErikS-web** [comment]
Will close this discussion. Pse continue under #118
